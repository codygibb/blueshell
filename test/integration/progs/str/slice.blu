s := "abcd"
print s[1:3]
print s[:3]
print s[1:]
print s[:-2]
print s[:-4]
print s[1:-4]
