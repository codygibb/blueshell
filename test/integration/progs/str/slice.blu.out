bc
abc
bcd
ab

error@7: Invalid_slice (1, -4)
